ko/admin/backup.inc

```diff
@@ -89,11 +89,11 @@ backupdb
 
         cubrid backupdb -t 4 demodb
 
-.. option:: -z, --compress
+.. option:: --no-compress
 
-    대상 데이터베이스를 압축하여 백업 파일에 저장한다. **-z** 옵션을 사용하면, 백업 파일의 크기 및 백업 시간을 단축시킬 수 있다. ::
+    기본적으로 큐브리드는 백업 볼륨의 크기를 줄이기 위해서, 대상 데이터베이스를 압축한 뒤 저장한다. 백업 파일 압축 실행 시 발생하는 CPU 사용량을 없애고 싶다면,  --no-compress 옵션을 설정해서 백업 볼륨 압축을 중지할 수 있다. ::
```

Comment:
```suggestion
    기본적으로 큐브리드는 백업 볼륨의 크기를 줄이기 위해서, 대상 데이터베이스를 압축한 뒤 저장한다. 백업 파일 압축 실행 시 발생하는 CPU 사용량을 없애고 싶다면,  --no-compress 옵션을 설정해서 백업 볼륨 압축하지 않을 수 있다. ::
```

Reply:
Thank you for the comment. I will apply it.